test/JDBC/expected/BABEL-5467-vu-verify.out

```diff
@@ -0,0 +1,524 @@
+numeric#!#numeric#!#numeric#!#numeric
+23456789.45#!#246913579.333332#!#445.11110#!#23457234.56110
+~~END~~
+
+
+select * from babel_5467_avgdata_8
+go
+~~START~~
+int#!#int#!#int#!#int
+2#!#2#!#2#!#2
+~~END~~
+
+
+select TABLE_CATALOG, TABLE_SCHEMA, TABLE_NAME, COLUMN_NAME, NUMERIC_PRECISION, NUMERIC_PRECISION_RADIX, NUMERIC_SCALE
+from information_schema.columns 
+where TABLE_NAME = 'babel_5467_avgdata_4' order by COLUMN_NAME
+go
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#tinyint#!#smallint#!#int
+master#!#dbo#!#babel_5467_avgdata_4#!#p#!#38#!#10#!#6
+master#!#dbo#!#babel_5467_avgdata_4#!#q#!#38#!#10#!#6
+master#!#dbo#!#babel_5467_avgdata_4#!#r#!#38#!#10#!#6
+master#!#dbo#!#babel_5467_avgdata_4#!#s#!#38#!#10#!#6
+~~END~~
+
+
+select TABLE_CATALOG, TABLE_SCHEMA, TABLE_NAME, COLUMN_NAME, NUMERIC_PRECISION, NUMERIC_PRECISION_RADIX, NUMERIC_SCALE
+from information_schema.columns 
+where TABLE_NAME = 'babel_5467_avgdata_5' order by COLUMN_NAME
+go
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#tinyint#!#smallint#!#int
+master#!#dbo#!#babel_5467_avgdata_5#!#p#!#10#!#10#!#2
+master#!#dbo#!#babel_5467_avgdata_5#!#q#!#15#!#10#!#6
+master#!#dbo#!#babel_5467_avgdata_5#!#r#!#8#!#10#!#5
+master#!#dbo#!#babel_5467_avgdata_5#!#s#!#14#!#10#!#5
+~~END~~
+
+
+select TABLE_CATALOG, TABLE_SCHEMA, TABLE_NAME, COLUMN_NAME, NUMERIC_PRECISION, NUMERIC_PRECISION_RADIX, NUMERIC_SCALE
+from information_schema.columns 
+where TABLE_NAME = 'babel_5467_avgdata_6' order by COLUMN_NAME
+go
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#tinyint#!#smallint#!#int
+master#!#dbo#!#babel_5467_avgdata_6#!#p#!#10#!#10#!#2
+master#!#dbo#!#babel_5467_avgdata_6#!#q#!#15#!#10#!#6
+master#!#dbo#!#babel_5467_avgdata_6#!#r#!#8#!#10#!#5
+master#!#dbo#!#babel_5467_avgdata_6#!#s#!#14#!#10#!#5
+~~END~~
+
+
+select TABLE_CATALOG, TABLE_SCHEMA, TABLE_NAME, COLUMN_NAME, NUMERIC_PRECISION, NUMERIC_PRECISION_RADIX, NUMERIC_SCALE
+from information_schema.columns 
+where TABLE_NAME = 'babel_5467_avgdata_7' order by COLUMN_NAME
+go
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#tinyint#!#smallint#!#int
+master#!#dbo#!#babel_5467_avgdata_7#!#p#!#38#!#10#!#2
+master#!#dbo#!#babel_5467_avgdata_7#!#q#!#38#!#10#!#6
+master#!#dbo#!#babel_5467_avgdata_7#!#r#!#38#!#10#!#5
+master#!#dbo#!#babel_5467_avgdata_7#!#s#!#38#!#10#!#5
+~~END~~
+
+
+select TABLE_CATALOG, TABLE_SCHEMA, TABLE_NAME, COLUMN_NAME, NUMERIC_PRECISION, NUMERIC_PRECISION_RADIX, NUMERIC_SCALE
+from information_schema.columns 
+where TABLE_NAME = 'babel_5467_avgdata_8' order by COLUMN_NAME
+go
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#tinyint#!#smallint#!#int
+master#!#dbo#!#babel_5467_avgdata_8#!#p#!#10#!#10#!#0
+master#!#dbo#!#babel_5467_avgdata_8#!#q#!#10#!#10#!#0
+master#!#dbo#!#babel_5467_avgdata_8#!#r#!#10#!#10#!#0
+master#!#dbo#!#babel_5467_avgdata_8#!#s#!#10#!#10#!#0
+~~END~~
+
+
+-- Tests for aggregates
+CREATE TABLE babel_5467_t5 (id integer PRIMARY KEY, amount decimal(38,6));
```

Comment:
Suggestion: We should also test variables along with aggregates

Reply:
Added tests for the same.